docs/api/1.x/index.rst

```diff
@@ -12,18 +12,22 @@ Full detailed API documentation:
    :maxdepth: 2
 
    authentication
+   resource
```

Comment:
My hope is that we will feel embarrassed by their exposure and that will make us fix them :) Also, other people looking at them might improve them, but if they're hidden, they probably won't be improved.